at_secondary/at_secondary_server/lib/src/notification/resource_manager.dart

```diff
@@ -110,7 +111,7 @@ class ResourceManager {
       //1. Adds errored notifications back to queue.
       _enqueueErrorList(errorList);
       //2. Calling close method to close the outbound connection
-      outBoundClient.outboundConnection.close();
+      outBoundClient.outboundConnection?.close();
```

Comment:
if outbound connection is null..this will throw NPE

Reply:
The conditional check after outboundConnection will protect from NPE.